samples/OpenIdConnect.AzureAdSample/Startup.cs

```diff
@@ -45,7 +45,7 @@ public void ConfigureServices(IServiceCollection services)
 
         public void Configure(IApplicationBuilder app, ILoggerFactory loggerfactory)
         {
-            loggerfactory.AddConsole(LogLevel.Information);
+            loggerfactory.AddConsole(Microsoft.Extensions.Logging.LogLevel.Information);
```

Comment:
Using both ADAL and LogLevel shouldn't be too common. It only happens in one of our templates.